hivemind/averaging/matchmaking.py

```diff
@@ -243,16 +243,16 @@ async def rpc_join_group(
         self, request: averaging_pb2.JoinRequest, _context: P2PContext
     ) -> AsyncIterator[averaging_pb2.MessageFromLeader]:
         """accept or reject a join request from another averager; if accepted, run him through allreduce steps"""
-        request_endpoint = None
+        request_peer_id = None
```

Comment:
nit: maybe just peer_id?

Reply:
Changed to `remote_peer_id` (to emphasize that it is not the local peer ID)